aws/identity/README.md

```diff
@@ -0,0 +1,143 @@
+
+### IAM Policies created
+
+| Policy Name                         | Description                                                               | Input Variable                    | Default |
+|-------------------------------------|---------------------------------------------------------------------------|-----------------------------------|---------|
+| `useExistingIamRolesPolicyDocument` | Allows IAM Users to attach or modify existing IAM Roles to AWS resources. | `defineUseExistingIamRolesPolicy` | `true`  |
+| `eksAdmins`                         | Administrative access to EKS.                                             | `defineEksAdminsPolicy`           | `true`  |
+
+## Stack Outputs
+
+Pulumi allows you to export the computed values of resources once they're finished initializing.
+This is useful, for example, for exporting IAM User credentials so that CI/CD systems can log in as
+that user.
+
+| Variable Name                  | Description                                                                                                                   |
+|--------------------------------|-------------------------------------------------------------------------------------------------------------------------------|
+| `networkCiUserAccessKeyId`     | The CI key ID for the infrastructure CI account. This can be used with the `aws` CLI tool to log in as this user account.     |
+| `networkCiUserAccessKeySecret` | The CI key secret for the infrastructure CI account. This can be used with the `aws` CLI tool to log in as this user account. |
+| `eksCiUserAccessKeyId`         | The CI key ID for the EKS CI account. This can be used with the `aws` CLI tool to log in as this user account.                |
+| `eksCiUserAccessKeySecret`     | The CI key secret for the EKS CI account. This can be used with the `aws` CLI tool to log in as this user account.            |
+| `kubeAppRoleArn`               | ARN of the Kubernetes App Role. This will be used by the CI system to deploy Kubernetes apps.                                 |
+
+## Background
+
+The AWS [Identity Access and Management (IAM)][iam] API allows AWS account administrators to manage
+permissions of user accounts and AWS workloads. In this section, we will discuss the basic set of
+nouns in the AWS IAM API which we use in this project.
+
+### Users
+
+Every human and machine must be assigned an [**IAM User**][users] account. IAM Users are the atoms of
+identity on AWS: a user account is associated with permissions that allow them to perform actions on
+AWS resoruces.
+
+A very commmon practice is to have a conventional naming scheme for user types. For example:
+`contractor.grace.hopper`, `bot.travisci`, `fte.barbara.liskov`, and so on. This application
+provides base classes `EmployeeUser`, `ContractorUser`, and `BotUser` to help enforce these naming
```

Comment:
To improve readability, let's maintain the order of the base classes inline with the order in the first set of examples you list out. Also, change `fte` to employee.

So lets have this:
scheme: `employee.barbara.liskov`,`contractor.grace.hopper`, and `bot.travisci`
classes: `EmployeeUser`, `ContractorUser`, and `BotUser`